### DISCUSSION TITLE
Github Actions

### DISCUSSION THREAD

**Discussion by ronnyteo** [community/community#52143] [Actions]
### Select Topic Area

Show & Tell

### Body

![Screenshot 2023-04-06 at 17 07 04](https://user-images.githubusercontent.com/78462114/230330190-3c82fd1a-ac20-4c2b-8e82-ad3cf61350e7.png)

Hi,

How to solve when using Github Actions when after each build, IMAGE_TAG: "8.0.0.0X_g66ab8e136" ID keep changing?

Thank you.

**Comment by airtower-luna** [comment]
I'm afraid the question is impossible to answer without more context. Where does the `IMAGE_TAG` come from, what is it used for? Best share the workflow, so people can see what you're trying to do.

FWIW, there are ways to define variables dynamically, see: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#environment-files

**Comment by ronnyteo** [comment]
how to get the git commit ID after checkin
